src/wkb2gltf.core.tests/MaterialCreatorTests.cs

```diff
@@ -29,20 +29,20 @@ public void MetallicRoughnessBaseColorTest()
     {
         // arrange
         var shader = new Shader();
-        // #008000 = green https://www.color-hex.com/color/008000 (0,128,0))
-        shader.PbrMetallicRoughness = new PbrMetallicRoughness() { BaseColor = "#008000", MetallicRoughness = "" };
+        // #008000 = green https://www.color-hex.com/color/008000 (0,128,0)), 81 as Alpha value
+        shader.PbrMetallicRoughness = new PbrMetallicRoughness() { BaseColor = "#81008000", MetallicRoughness = "" };
```

Comment:
a check is this correct hex code 81008000 ? got invalid on that color-hex site

Reply:
see other comment about invalid hex